src/Features/Core/Portable/Completion/Providers/AbstractKeywordCompletionProvider.cs

```diff
@@ -77,7 +77,8 @@ protected virtual CompletionItem CreateItem(RecommendedKeyword keyword, TextSpan
                 filterSpan: filterSpan,
                 descriptionFactory: c => Task.FromResult(keyword.DescriptionFactory(c)),
                 glyph: Glyph.Keyword,
-                isIntrinsic: keyword.IsIntrinsic);
+                isIntrinsic: keyword.IsIntrinsic,
+                preselect: keyword.ShouldPreselect);
```

Comment:
What does it mean for two of these to preselect? We preselect the first in the list, and filtering already treats it as priority?